sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_operations.py

```diff
@@ -274,7 +280,34 @@ def begin_create(
             except HttpResponseError:
                 module_logger.warning("Failed to get parent hub for project, some values won't be transferred:")
         try:
-            return super().begin_create(workspace, update_dependent_resources=update_dependent_resources, **kwargs)
+            poller = super().begin_create(workspace, update_dependent_resources=update_dependent_resources, **kwargs)
+            # ai_services set - try to wait for workspace operation to complete, then make connection.
+            if ai_services is not None:
+                try:
+                    print(
+                        "Creating connection to AI Services resource. "
+                        + "This requires waiting for the workspace operation to complete..."
+                    )
+                    poller.result()
+                    from azure.ai.ml import MLClient
+                    from azure.ai.ml.entities import AzureAIServicesConnection
+
+                    ai_services_name = ai_services.split("/")[-1]
+                    ai_services_target = f"https://{ai_services_name}.cognitiveservices.azure.com/"
+                    sub_client = MLClient(
+                        subscription_id=self._subscription_id,
+                        resource_group_name=workspace.resource_group,
+                        credential=self._credentials,
+                        workspace_name=workspace.name,
+                    )
+                    conn = AzureAIServicesConnection(
+                        name="Default_AIServices", endpoint=ai_services_target, ai_services_resource_id=ai_services
+                    )
+                    sub_client.connections.create_or_update(connection=conn)
+                except HttpResponseError as e:
+                    print(e)
```

Comment:
can you validate what `print(e)` actually does? If we're not raising an exception, you probably don't want to confuse the customer if the `print` then dumps out a stack trace. 

also like with the above comment, consider using the `module_logger` and using `warning` since this is an exception case

Reply:
also, you probably just want to extract out the error message from `e` and log just one message saying creating the connection failed + the error message (if you're not meaning to raise an exception here)

Reply:
Removing as per discussion in other PR.